core/sail/shacl/src/main/java/org/eclipse/rdf4j/sail/shacl/AST/PropertyShape.java

```diff
@@ -58,6 +60,10 @@
 		this.parent = parent;
 	}
 
+	public Value getEXP() {
+		return this.Expected;
+	}
+
```

Comment:
A comment I have above about maybe making an interface method. Also I prefer full names for things. eg. getExpected().